.git-blame-ignore-revs

```diff
@@ -0,0 +1,12 @@
+# When making commits that are strictly formatting/style changes, add the
+# commit hash here, so git blame can ignore the change. See docs for more
+# details:
+# https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
+#
+# Github will use this file to ignore these commits for blame.
+#
+# Run this to configure git locally to also ignore these commits:
+#   git config blame.ignoreRevsFile .git-blame-ignore-revs
+
+# Ran prettier on the entire codebase
+7dd26612601bbbcfccb81fdf939b7b1a301f1366
```

Comment:
Is this going to work after this PR gets squashed and merged?

Reply:
Or we just omit this part for now and/or update it after merging to be the squashed commit. We can ignore this entire PR.

Reply:
That works too!